image.pyspark-py36.yaml

```diff
@@ -0,0 +1,48 @@
+schema_version: 1
+
+name: "radanalyticsio/radanalytics-pyspark-py36"
+version: 1.0
+release: dev
+from: "centos/python-36-centos7:latest"
+labels:
+  - name: "io.k8s.description"
+    value: "Platform for building a radanalytics pyspark app"
```

Comment:
Do we want to include 36 in this description too, or do you think that's overkill?

Reply:
i think that would be helpful, if only so folks know what they made